keanu-project/src/test/java/io/improbable/keanu/e2e/foodpoisoning/FoodPoisoningTest.java

```diff
@@ -76,12 +79,18 @@ public NetworkSamples sample(int n) {
         return MetropolisHastings.withDefaultConfig(random).getPosteriorSamples(myNet, myNet.getLatentVertices(), n);
     }
 
-    public void generateSurveyData(int peopleCount, boolean oystersAreInfected, boolean lambIsInfected, boolean toiletIsInfected) {
+    public void generateSurveyData(int peopleCount, boolean oystersAreInfected, boolean lambIsInfected, boolean toiletIsInfected) throws VertexLabelException {
```

Comment:
I feel like theres a tensor way to do this but it's nice to have it as an example how to use plates.